recipes-core/initrdscripts/initramfs-framework/bootfs

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+# Copyright (C) 2018 Apertus Solutions, LLC
+# Licensed on MIT
+
+bootfs_enabled() {
+	if [ -n "${bootparam_boot}" ]; then
+		return 0
+	else
+		return 1
+	fi
+}
+
+bootfs_run() {
+	BOOTFS_DIR="/${ROOTFS_DIR}/boot/system"
+        if [ -z "$BOOTFS_DIR" ]; then
+                return
+        fi
+        C=0
+	# will use rootfs' delay and timeout values
+        delay=${bootparam_rootdelay:-1}
+        timeout=${bootparam_roottimeout:-5}
+        while [ ! -d $BOOTFS_DIR/grub ]; do
+                if [ $(( $C * $delay )) -gt $timeout ]; then
+                        fatal "boot '$bootparam_boot' doesn't exist or does not contain a /grub."
+                fi
+
+                if [ -n "$bootparam_boot" ]; then
+                        debug "No e2fs compatible filesystem has been mounted, mounting $bootparam_boot..."
+
+                        if [ "`echo ${bootparam_boot} | cut -c1-5`" = "UUID=" ]; then
```

Comment:
I would recommend swapping the backticks for $() evaluation syntax here and elsewhere in the PR if possible.  I think most modern bash is preferring the POSIX-style nowadays.

Reply:
Yes, I generally use $() than ` but this is a duplication from upstreams rootfs module with ROOTFS replaced everywhere with BOOTFS. BTW, POSIX defines both as legitimate.[1](http://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html)

Reply:
Or without a subshell, pipe and binary call: `[ "${bootparam_boot:0:5}" = "UUID=" ]` seems to work fine with /bin/sh in OpenXT.
Maybe a bit picky, and maybe too bash-specific, but there are so many subshells with pipes and binary calls in this PR that it could be worth it to even switch to /bin/bash...